docs/support.md

```diff
@@ -0,0 +1,148 @@
+(support-features)=
+# Support Features
+
+The package bundles a few support and utility functions that try to fill a few
+gaps you will observe when working with CrateDB, a distributed OLAP database,
+since it lacks certain features, usually found in traditional OLTP databases.
+
+A few of the features outlined below are referred to as [polyfills], and
+emulate a few functionalities, for example, to satisfy compatibility issues on
+downstream frameworks or test suites. You can use them at your disposal, but
+you should know what you are doing, as some of them can seriously impact the
+performance.
+
+
+(support-automatic-refresh)=
+## Automatic Table REFRESH after DML
+
+:::{rubric} Introduction
+:::
+CrateDB is [eventually consistent]. Data written with a former statement is
+not guaranteed to be fetched with the next following select statement for the
+affected rows.
+
+Data written to CrateDB is flushed periodically, the refresh interval is
+1000 milliseconds by default, and can be changed. More details can be found in
+the reference documentation about [table refreshing](inv:crate-reference#refresh_data).
+
+There are situations where stronger consistency is required, for example when
+needing to satisfy test suites of 3rd party frameworks, which usually do not
+take such special behavior of CrateDB into consideration. 
+
+:::{rubric} Utility
+:::
+The `refresh_after_dml` utility will configure an SQLAlchemy engine or session
+to automatically invoke relevant `REFRESH TABLE` statements after each DML
+operation (INSERT, UPDATE, DELETE), for the corresponding entities / tables.
+
+```python
+import sqlalchemy as sa
+from sqlalchemy_cratedb.support import refresh_after_dml
+
+engine = sa.create_engine("crate://")
+refresh_after_dml(engine)
+```
+
+```python
+import sqlalchemy as sa
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy_cratedb.support import refresh_after_dml
+
+engine = sa.create_engine("crate://")
+session = sessionmaker(bind=engine)()
+refresh_after_dml(session)
+```
+
+:::{warning}
+Refreshing the table after each DML operation can cause serious
+performance degradation, and should only be used on low-volume, low-traffic
+data, when applicable, and if you know what you are doing.
+:::
+
+
+## Support for pandas and Dask
+Todo.
```

Comment:
Maybe remove this if you're not planning to fill in this PR.

Reply:
I've just updated the patch, with this section completed.

-- https://sqlalchemy-cratedb--28.org.readthedocs.build/support.html#support-for-pandas-and-dask